src/sparseml/transformers/sparsification/obcq/example_llama.yaml

```diff
@@ -50,8 +50,7 @@ test_stage:
       sequential_update: False
       quantize: True
       percdamp: 0.01
-      prunen: 0
-      prunem: 0
+      mask_structure: "0:0"
```

Comment:
Why were these changed?

Reply:
This is the intended change; to move away from specifying prunen and prunem separately; as a future PR this mask_structure attribute can also be used to do more nifty things, like 4-block/channel/filter level masks